src/webview_provider/docsEditPanel.ts

```
@@ -465,6 +477,46 @@
+      doc.delete(key);
+    }
+  }
+
+  private findEntityInParsedDoc(
+    models:
+      | YAMLSeq<DocumentationSchema["models"]["0"]>
+      | YAMLSeq<DocumentationSchemaColumn>
+      | undefined,
+    predicate: (name: string) => boolean,
+  ) {
+    if (models && models.items) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)